src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -90,6 +90,6 @@ public CommandResult execute(Model model) throws CommandException {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof AddCommand // instanceof handles nulls
-                && toAdd.equals(((AddCommand) other).toAdd));
+                && toAdd.equals(((AddCommand) other).toAdd) && personType.equals(((AddCommand) other).personType));
```

Comment:
I think just for consistency sake you can put the `&& personType.equals(((AddCommand) other).personType))` on a new line.